resources/web/README.md

```diff
@@ -1,15 +1,69 @@
-
 ### Frontend Web Developer Roadmap
 
 <LevelWithButton :imageRight="false"  desc="A community-created roadmap for modern frontend web development." image="/images/frontend-roadmap.png" button="Check it out!" link="https://roadmap.sh/frontend" />
 
 ### Backend Web Developer Roadmap
 
 <LevelWithButton desc="A community-created roadmap for modern backend web development." image="/images/backend-roadmap.png" button="Check it out!" link="https://roadmap.sh/backend" />
+
+## HTML, CSS & JavaScript Cheat sheets!
+<div class="scrolling-wrapper">
+<div class="scroll-child">
+<iframe src="https://lifeyourway.net/printables/blogging-html-cheat-sheet.pdf" width="550px" height="500px"></iframe>
+</div>
+<div class="scroll-child">
+<iframe src="https://courses.cs.washington.edu/courses/cse154/16sp/cheat-sheets/css-cheat-sheet.pdf" width="580px" height="500px"></iframe>
+</div>
+  <div class="scroll-child">
+   <iframe src="https://cheatography.com/davechild/cheat-sheets/javascript/pdf/" width="600px" height="500px"></iframe>
+  </div>
+</div>
+
+<style scoped>                                                                                                          
+.scrolling-wrapper {                                                                                                      
+display: flex;                                                                                                          
+flex-wrap: nowrap;                                                                                                      
+overflow-x: auto;                                                                                                                                                                                                                             
+}                                                                                                                       
+.scroll-child {                                                                                                             
+flex: 0 0 auto;                                                                                                        
+ margin-right: 36px;                                                                                                     
+font-size: 24px                                                                                                       
+}                                                                                                                     
+</style>                                                                                                                                 
+
+## Some Popular Web Development tools 
```

Comment:
There are collections out there that are more comprehensive than the one you've listed. 

https://medium.com/javarevisited/10-of-the-most-popular-javascript-frameworks-libraries-for-web-development-in-2019-a2c8cea68094
https://www.freecodecamp.org/news/handy-web-development-toolkit/

Mentioning developer tools and linking an overview of that would be useful.